public/test/specs/time_srv_specs.js

```diff
@@ -78,17 +78,24 @@ define([
     });
 
     describe('setTime', function() {
-      it('should return disable refresh for absolute times', function() {
+      it('should return disable refresh if refresh is disabled for any range', function() {
         _dashboard.refresh = false;
 
         ctx.service.setTime({from: '2011-01-01', to: '2015-01-01' });
         expect(_dashboard.refresh).to.be(false);
       });
 
+      it('should restore refresh for absolute time range', function() {
+        _dashboard.refresh = '30s';
+
+        ctx.service.setTime({from: '2011-01-01', to: '2015-01-01' });
+        expect(_dashboard.refresh).to.be('30s');
+      });
+
       it('should restore refresh after relative time range is set', function() {
         _dashboard.refresh = '10s';
         ctx.service.setTime({from: moment([2011,1,1]), to: moment([2015,1,1])});
-        expect(_dashboard.refresh).to.be(false);
+        expect(_dashboard.refresh).to.be('10s');
```

Comment:
Do we want to disable the auto-refresh when we zoom in? Because zooming in will automatically set the time range to absolute time range. And in this PR we want to enable auto-refresh for the absolute time range.

So, I think zooming in should retain the refresh interval. Hence, expecting "10s" amd not "false". Also, this feature used to exist in older versions.